tests/flow/test_v7_encode_decode.py

```diff
@@ -0,0 +1,137 @@
+from base import FlowTestsBase
+import os
+import sys
+from RLTest import Env
+from redisgraph import Graph, Node, Edge
+
+redis_con = None
+
+class test_v7_encode_decode(FlowTestsBase):
+    def __init__(self):
+        self.env = Env(moduleArgs='ENTITIES_THRESHOLD 10')
+        global redis_con
+        redis_con = self.env.getConnection()
+        info = redis_con.execute_command("INFO")
+        version = info['redis_version'].split('.')
+        major = int(version[0])
+        minor = int(version[1])
+        # Look for redis 6 RC versions and up
+        if major < 5 or major == 5 and minor < 9:
+            self.env.skip()
+
+    def test01_nodes_over_multiple_keys(self):
+        graph_name = "nodes_over_multiple_keys"
+        redis_graph = Graph(graph_name, redis_con)
+        # Create 3 nodes meta keys
+        redis_graph.query("UNWIND range(0,20) as i CREATE ({val:i})")
```

Comment:
How come creating 20 nodes will result in 3 meta keys?
Is that due to `ENTITIES_THRESHOLD` equal 10?
Can we omit `self.env = Env(moduleArgs='ENTITIES_THRESHOLD 10')` and use the default?

Reply:
it is 21 nodes [0-21] so it will create 3 meta keys since each key contains 3 entities.
the default is 100,000 entities and the Valgrind tests take forever